packages/devtools/src/components/Card/Medias/TextBlockCard.tsx

```diff
@@ -1,24 +1,21 @@
-import * as marked from 'marked';
-
 import { TextBlock } from '@teams.sdk/cards';
 import { useEffect, useState } from 'react';
+import { MarkdownContent } from '../../MarkdownContent';
 
 export interface TextBlockCardProps {
   readonly value: TextBlock;
 }
 
-export default function ImageCard({ value }: TextBlockCardProps) {
+export default function TextBlockCard({ value }: TextBlockCardProps) {
   const [html, setHtml] = useState<string>();
 
   useEffect(() => {
-    setHtml(marked.parse(value.text, { async: false, gfm: true }));
+    setHtml(value.text);
```

Comment:
`useMemo` is likely better for this, and more readable. this just means that `html` is always one render behind.

Reply:
Logged for fast-follow https://github.com/microsoft/spark.js/issues/18